src/Morphic-Widgets-FastTable/FTMultipleSelection.class.st

```diff
@@ -14,51 +14,43 @@ FTMultipleSelection >> isMultiple [
 ]
 
 { #category : #private }
-FTMultipleSelection >> selectAppendingRowIndex: rowIndex [
+FTMultipleSelection >> selectAppendingIndex: index [
 	| currentSelection newSelection |
-	currentSelection := self table selectedRowIndexes.
+	currentSelection := self table selectedIndexes.
 	
-	newSelection := (currentSelection includes: rowIndex)
-		ifTrue: [ currentSelection copyWithout: rowIndex ]
-		ifFalse: [ currentSelection copyWithFirst: rowIndex].
+	newSelection := (currentSelection includes: index)
+		ifTrue: [ currentSelection copyWithout: index ]
+		ifFalse: [ currentSelection copyWithFirst: index].
 		
-	self table selectRowIndexes: newSelection
+	self table selectIndexes: newSelection
 ]
 
 { #category : #accessing }
-FTMultipleSelection >> selectRowIndex: rowIndex event: event [
-	event shiftPressed ifTrue: [ ^ self selectRowIndexesUpTo: rowIndex ].
-	event commandKeyPressed ifTrue: [ ^ self selectAppendingRowIndex: rowIndex ].
-	self toggleRowIndex: rowIndex
-
+FTMultipleSelection >> selectIndex: index event: event [
+	event shiftPressed ifTrue: [ ^ self selectIndexesUpTo: index ].
+	OSPlatform current isMacOS
```

Comment:
Looks like this should be a method on the current platform, OSPlatform current appendSelectionFrom: event or so?

Reply:
I already opened an issue about that.